src/main/java/modify_event_use_case/ModifyEventDsGateway.java

```diff
@@ -0,0 +1,10 @@
+package modify_event_use_case;
+
+import java.time.LocalTime;
+
+public interface ModifyEventDsGateway {
+    boolean titleExistsInDay(int dayIndex, String title);
+
+    boolean isTimeConflict(int dayIndex, String title, LocalTime newStart, LocalTime newEnd);
+    void save(ModifyEventDsInputData dataAccessInput);
```

Comment:
Maybe add a newline after line 8? It looks odd currently.

Reply:
It does look odd... I'll add a line.